force-app/main/default/triggers/WorkBreakdownTrigger.trigger

```diff
@@ -0,0 +1,23 @@
+trigger WorkBreakdownTrigger on Work_Breakdown_Structure__c(before update, after update) {
+	if (Trigger.isBefore) {
+		for (Work_Breakdown_Structure__c wbs : Trigger.New) {
+			if (wbs.Status__c == Constants.WBS_STATUS_SENT && wbs.Opportunity__c != null) {
+				wbs.IsSynced__c = true;
+			}
+		}
+	} else if (Trigger.isAfter) {
+		for (Work_Breakdown_Structure__c wbs : Trigger.New) {
+			if (wbs.Status__c == Constants.WBS_STATUS_SENT && wbs.Opportunity__c != null) {
+				Opportunity opp = [
+					SELECT Amount, Decision_Maker__r.LastName
+					FROM Opportunity
+					WHERE Id = :wbs.Opportunity__c
+				];
+				opp.Amount = wbs.Total_Amount__c;
+				opp.Work_Breakdown_Structure__c = wbs.Id;
+				EmailUtility.toSendEmailWithTemplate(opp.Decision_Maker__r.LastName, Contact.SObjectType);
```

Comment:
email sendsin for loop (limit in one transaction):
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_gov_limits.htm
![image](https://user-images.githubusercontent.com/23176834/133984981-d58ab28e-ef27-43c6-8242-2e2cf3a5bce2.png)